core/src/test/java/org/apache/struts2/util/StrutsUtilTest.java

```diff
@@ -145,28 +147,51 @@ public void testGetContextMethod() {
     }
 
 
-    public void testMakeSelectListMethod() {
-        String[] selectedList = new String[] { "Car", "Airplane", "Bus" };
-        List list = new ArrayList();
-        list.add("Lorry");
-        list.add("Car");
-        list.add("Helicopter");
+    public void testMakeSelectList() {
+        String[] selectedList = new String[]{"Car", "Airplane", "Bus"};
+        List<String> list = Arrays.asList("Lorry", "Car", "Helicopter");
+
+        stack.getContext().put("mySelectedList", selectedList);
+        stack.getContext().put("myList", list);
+
+        List<ListEntry> listMade = strutsUtil.makeSelectList("#mySelectedList", "#myList", null, null);
+
+        assertEquals(listMade.size(), 3);
```

Comment:
for assertEquals, the expected value should be the first argument

Reply:
Seems all the assertions in this class are mis-ordered, I've amended the ones I've touched :)